Skeleton/src/Order.cpp

```diff
@@ -21,10 +62,11 @@ string Order::getStatusString() const {
             return "COMPLETED";
     }
 }
-
-void Order::setCollectorId(int collectorId) {collectorId = collectorId;}
-void Order::setDriverId(int driverId){driverId = driverId;}
-int Order::getCollectorId() const {return collectorId;}
-int Order::getDriverId() const {return driverId;}
-OrderStatus Order::getStatus() const {return status;}
-const string Order::toString() const {return "Order status";} 
\ No newline at end of file
+const string Order::toString() const
+{
+    return "Order ID: " + std::to_string(getId())
```

Comment:
you are in the class so you can call the private parameters without the getters

Reply:
resolved in my new branch